### PR TITLE
Use the new media types for oci

### LINKED ISSUE #196: OCI: use v0.5.0 media-types
OCI v0.5.0 will change media-types. This is unreleased but is likely to be released in the next 7-10 days.

### PR DESCRIPTION
**Pull request by vishvananda**
Fixes #196

### CODE REVIEW

**Comment by lucab** [comment]
This LGTM, but I'd defer OCI topics to @jonboulle.

File: lib/internal/typesV2/docker_types.go [hunk 1]
```diff
@@ -31,9 +31,8 @@ const (
 
 	MediaTypeOCIManifest     = "application/vnd.oci.image.manifest.v1+json"
 	MediaTypeOCIManifestList = "application/vnd.oci.image.manifest.list.v1+json"
-	MediaTypeOCIConfig       = "application/vnd.oci.image.serialization.config.v1+json"
-	MediaTypeOCIRootFS       = "application/vnd.oci.image.serialization.rootfs.tar.gzip"
-	MediaTypeOCICombined     = "application/vnd.oci.image.serialization.combined.v1+json"
+	MediaTypeOCIConfig       = "application/vnd.oci.image.config.v1+json"
+	MediaTypeOCILayer        = "application/vnd.oci.image.layer.tar+gzip"
```
Comment (jonboulle) [inline lib/internal/typesV2/docker_types.go]: could you update this one please?
https://github.com/opencontainers/image-spec/blame/master/media-types.md#L9

Comment (lucab) [inline lib/internal/typesV2/docker_types.go]: I think the idea - as per https://github.com/appc/docker2aci/issues/196#issuecomment-255408394) - is to update this again once -rc2 is out.

Comment (jonboulle) [inline lib/internal/typesV2/docker_types.go]: Good catch.

**Pull request merged**